config/manager/manager.yaml

```diff
@@ -31,6 +32,9 @@ spec:
         - containerPort: 9440
           name: healthz
           protocol: TCP
+        - containerPort: 8443
```

Comment:
this should be as well `${CAPG_DIAGNOSTICS_ADDRESS:=:8443}` or no?

Reply:
Good point 👍